### ISSUE TITLE
Unable to access the top flyout button for mage Teleport/Portal

### ISSUE THREAD

**Issue by raysmith59** [brittyazel/Neuron#264]
This was an issue a while back (year or two) but was fixed, seems to be back.  When I move my mouse up the list of flyout buttons it's fine right until I'm about to get to the one at the top at which point the flyout closes.

Hunter's Pet Utility flyout has a somewhat similar issue where if I move to the top half of the top button the flyout closes.

Possibly an issue where the height of the flyout buttons isn't being correctly determined, all flyouts seem to be affected but the more buttons, the worse it is.

**Comment by brittyazel** [comment]
Yeah this might be a compounding issue with the button area being miscalculated. I'll take a look at the flyout code where we compute that and see if anything seems awry

**Comment by brittyazel** [comment]
Is true for both with and without Masque? I was trying to test this earlier with my hunter, the call pet ability only has 5 total buttons. Regardless, I wasn't having the "top half of the button" issue, but then again, I was using Masque at the time. I'll test again without masque

**Comment by raysmith59** [comment]
Thought I responded to this but no, I don't use Masque, far as I know I don't have anything that interacts with Neuron aside from maybe OmniCC.

**Comment by raysmith59** [comment]
I hadn't checked on this issue in a while and at this point I can get to the top of the portal list so the issue seems to be resolved, closing.

**Comment by raysmith59** [comment]
Just checked again and now I can't get to the top flyout buttons again for mage portals or teleports, re-opening.

**Comment by raysmith59** [comment]
I've also determined that if I set the bar scale to 1 it works fine, my bars are at scale 1.25 and at that point it doesn't work.  So, it's possible the calculation isn't factoring in scale properly.
